src/modules/UI/scenes/TransactionList/TransactionList.ui.js

```diff
@@ -47,25 +48,23 @@ export type StateProps = {
   contacts: ContactsState,
   fiatSymbol: string,
   showToWalletModal: boolean,
-  requiredConfirmations?: number
+  requiredConfirmations?: number,
+  isAccountBalanceVisible: boolean
 }
 
 export type DispatchProps = {
-  fetchMoreTransactions: (walletId: string, currencyCode: string, reset: boolean) => any
+  fetchMoreTransactions: (walletId: string, currencyCode: string, reset: boolean) => any,
+  toggleAccountBalanceVisibility: () => void
 }
 
 type Props = StateProps & DispatchProps
 
 type State = {
   focused: boolean,
-  balanceBoxVisible: boolean,
   op: any,
   animation: any,
-  balanceBoxOpacity: any,
-  balanceBoxHeight: any,
   width: ?number,
-  reset: boolean,
-  showBalance: boolean
```

Comment:
why did you get rid of these?

Reply:
These state variables are related to defunct Search Bar that we decided not to implement (yet) approximately 6 months ago. They are legacy code that are inert.